src/test/java/com/google/step/TestConstants.java

```diff
@@ -127,4 +127,28 @@
           USER_ID_B,
           RESTAURANT_ID_B,
           TIME_A);
+
+  public static final String DEAL_A_BRIEF_JSON =
+      String.format(
+          "{"
+              + "    \"id\": %d,"
+              + "    \"description\": \"%s\","
+              + "    \"image\": \"%s\","
+              + "    \"restaurant\": %d,"
+              + "    \"votes\": %d,"
+              + "    \"poster\": %d"
```

Comment:
Are we removing votes, restaurants and poster from deal brief json?

Reply:
oyea i think we are. Which PR is it in?